keras/layers/convolutional.py

```diff
@@ -343,6 +343,194 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
+class Convolution3D(Layer):
+    '''Convolution operator for filtering windows of three-dimensional inputs.
+    When using this layer as the first layer in a model,
+    provide the keyword argument `input_shape`
+    (tuple of integers, does not include the sample axis),
+    e.g. `input_shape=(3, 10, 128, 128)` for 10 frames of 128x128 RGB pictures.
+
+    Note: this layer will only work with Theano for the time being.
+
+    # Input shape
+        5D tensor with shape:
+        `(samples, channels, len_input_dim1, len_input_dim2, len_input_dim3)` if dim_ordering='th'
```

Comment:
I think there should be a way to distinguish the input's dimensions vs. the kernel's dimensions. Convolution2D, for example, uses `rows` for the input and `nb_row` for the kernel. Would you say it's clear enough for Convolution3D to just use `conv_dim*` for both cases?